src/app/cube/library/components/notification-card/notification-card.component.html

```diff
@@ -1,9 +1,10 @@
 <div class='notification-card'>
-  <button aria-label='delete notification' class='notification-card__exit-button' (activate)='onDeleteButtonClick($event)'>
+  <button aria-label='Dismiss notification' class='notification-card__exit-button' (activate)='onDeleteButtonClick($event)' tip="Dismiss notification"
+  tipPosition="top">
     <i class="fal fa-times"></i>
   </button>
   <h4 class='notification-card__title'>
-    {{ (notification.attributes?.name.length>30) ? (notification.attributes?.name | slice:0:30)+'...' : notification.attributes?.name }}
+    {{ (notification.attributes?.name.length>25) ? (notification.attributes?.name | slice:0:22)+'...' : notification.attributes?.name }}
```

Comment:
Why are these different values?

Reply:
I changed something in the css to better fit the cards in the container when it resizes and needed five less characters to make it work.